cmd/mt-parrot/monitor.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"fmt"
+	"github.com/grafana/metrictank/stacktest/graphite"
+	"github.com/grafana/metrictank/stats"
+	log "github.com/sirupsen/logrus"
+	"math"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+var (
+	httpError           = stats.NewCounter32("parrot.monitoring.error;error=http")
+	decodeError         = stats.NewCounter32("parrot.monitoring.error;error=decode")
+	parsePartitionError = stats.NewCounter32("parrot.monitoring.error;error=parsePartition")
```

Comment:
this one seems oddly too specific.
Note that the response body should contain num_partitions series entries, each of which named by a number, covering each partition exactly once.  We should have 1 error type that just covers all of these potential problems.
maybe `error=faulty` (i'm not feeling very inspired right now)

Reply:
how about `error=invalid`? 43ea289d12886de8632aa127528c89b7c4360cc8

Reply:
invalid sounds good. but my comment was only about the last stat.
It was *good* to have the decode error separate from the 3rd error

Reply:
woops, back in ed4457c896fd36656260b8f50b3c4e2762df3402